fieldservice_sale_recurring/models/sale_order_line.py

```diff
@@ -20,21 +20,24 @@ def _field_create_fsm_recurring_prepare_values(self):
         self.ensure_one()
         template = self.product_id.fsm_recurring_template_id
         product = self.product_id
+        order_template = product.fsm_order_template_id or template.fsm_order_template_id
         note = self.name
         if template.description:
             note += "\n " + template.description
-        return {
+        vals = {
             "location_id": self.order_id.fsm_location_id.id,
             "start_date": self.order_id.expected_date,
             "fsm_recurring_template_id": template.id,
             "description": note,
             "max_orders": template.max_orders,
             "fsm_frequency_set_id": template.fsm_frequency_set_id.id,
-            "fsm_order_template_id": product.fsm_order_template_id.id
-            or template.fsm_order_template_id.id,
+            "fsm_order_template_id": order_template.id,
```

Comment:
```suggestion
            "fsm_order_template_id": order_template.id,
            "team_id": order_template.team_id.id or False,
```

Reply:
See previous comment. The same applies for the `fsm.recurring` model, whose `team_id` is required.